README.md

```diff
@@ -67,6 +67,16 @@ Winit provides the following features, which can be enabled in your `Cargo.toml`
 * `wayland` (enabled by default): On Unix platform, compiles with the Wayland backend
 * `mint`: Enables mint (math interoperability standard types) conversions.
 
+## MSRV Policy
+
+The Minimum Supported Rust Version (MSRV) of this crate is **1.65**. Changes to the MSRV will be released as patch version updates to `winit`. Once `winit` v1.0 is released, they will instead be released as minor version updates.
```

Comment:
This paragraph is redundant. You can't change msrv without releasing a crate.

Reply:
This paragraph specifies what kind of release we require for a MSRV change. We were still actively discussing it in https://github.com/rust-windowing/winit/pull/3046#discussion_r1305838780.

Reply:
I mean, msrv bump is not a breaking change according to rust ecosystem. Though, you could just say that patch release can't bump MSVR, because that's how it's done today and that's how it should be done, because downstream could pull patches in, however patch releases should be delivered to distros without a need to bump a rust version for the patch, so it's more rolling.